eitprocessing/datahandling/sparsedata.py

```diff
@@ -13,38 +13,40 @@
 
 @dataclass(eq=False)
 class SparseData(Equivalence, SelectByTime):
-    """Container for data occuring at unpredictable time points.
+    """Container for data related to individual time points.
 
-    In sparse data the time points are not necessarily evenly spaced. Data can consist time-value pairs or only time
-    points. Values generally are numeric values in arrays, but can also be lists of different types of object.
+    Sparse data is data for which the time points are not necessarily evenly spaced. Data can consist time-value pairs
+    or only time points. Values generally are numeric values in arrays, but can also be lists of different types of
+    object.
 
-    Sparse data differs from IntervalData in that each data points is associated with a single time point rather than a
+    Sparse data is assumed to be sequential (i.e. only a single data points at a single time point, and sorted by time).
+    Sparse data can contain zero or more time points/time-values pairs.
```

Comment:
A few comments:
- I think it would be good to either enforce that it is sequential or not comment on it (idem for interval data).
- The part between brackets is unclear to me.
- I think the last sentence can be omitted (if not, it should be included for the other containers as well).

Reply:
I removed both sentences. 

We currently have almost no checks for values that are set. This might be useful to do at some point, but not part of this PR. 

I previously added the second sentence because you commented you weren't sure whether SparseData was a single or more than one time point. It can be any number of time points.

Reply:
> I previously added the second sentence because you commented you weren't sure whether SparseData was a single or more than one time point. It can be any number of time points.

I understand. I think that's my bad. I think it's more a generic confusion I keep reverting to and then remembering regarding data collections and data containers and data sets rather than about sparse data per se. It would be good to at some point create clear documentation defining what each of these is (for human comprehension).

Reply:
> We currently have almost no checks for values that are set. This might be useful to do at some point, but not part of this PR.

Agreed. I've added a comment to [#208 comment](https://github.com/EIT-ALIVE/eitprocessing/issues/208#issuecomment-2149397241)